### PR TITLE
Bug/several mail fixes

### PR DESCRIPTION
**Pull request by kalikaneko**

### CODE REVIEW

File: src/leap/mail/mail.py [hunk 1]
```diff
@@ -744,7 +746,7 @@ def __cb_signal_unread_to_ui(self, unseen):
         :param unseen: number of unseen messages.
         :type unseen: int
         """
-        emit_async(catalog.MAIL_UNREAD_MESSAGES, str(unseen))
+        emit_async(catalog.MAIL_UNREAD_MESSAGES, self.store.uuid, str(unseen))
```
Comment (meskio) [inline src/leap/mail/mail.py]: That will require changes in `bitmask_client/src/leap/bitmask/gui/mail_status.py` I don't think `_mail_handle_imap_events_slot` does handle the uuid. And probably we should update the comment at `leap_pycommon/src/leap/common/events/catalog.py` describing correctly the usage of this event.

Comment (kalikaneko) [inline src/leap/mail/mail.py]: mail_status was already changed

Comment (kalikaneko) [inline src/leap/mail/mail.py]: common change: https://github.com/leapcode/leap_pycommon/pull/140

Comment (meskio) [inline src/leap/mail/mail.py]: I don't understand how mail_status handle the uuid, but I will trust you that this is fine and go ahead with it.

**Pull request merged**